server/app/views/applicant/NorthStarApplicantProgramSummaryView.java

```diff
@@ -66,6 +66,11 @@ public String render(Request request, Params params) {
     context.setVariable("notEligibleBannerMessage", params.notEligibleBannerMessage());
     context.setVariable("errorBannerMessage", request.flash().get(FlashKey.ERROR));
 
+    // Modals
+    String duplicateFlashValue = request.flash().get(FlashKey.DUPLICATE_SUBMISSION).orElse("");
```

Comment:
**nitpick:** Noticed here (https://github.com/civiform/civiform/blob/fafb62db776ce51d001b34507de341c8260352c8/server/app/views/applicant/NorthStarApplicantProgramSummaryView.java#L73-L75) we use an optional string. We could consider making this consistent with that.